### PR TITLE
Update project to 3.4

### PR DESCRIPTION
**Pull request by HaSa1002**
This PR updates the project to 3.4. There have been some minor scene file changes.

### CODE REVIEW

File: src/Resources/SignalTypes/Hp/Form/HpForm.tscn [hunk 1]
```diff
@@ -117,10 +117,6 @@ script = ExtResource( 2 )
 [node name="formsignal_hp" parent="." instance=ExtResource( 1 )]
 transform = Transform( 1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 1.667 )
 
-[node name="Skeleton" parent="formsignal_hp/Armature" index="0"]
-bones/0/bound_children = [ NodePath("lowerLightAttachment") ]
-bones/1/bound_children = [ NodePath("upperLightAttachment") ]
-
```
Comment (DasCapschen) [inline src/Resources/SignalTypes/Hp/Form/HpForm.tscn]: This might be problematic... will need to check if is still works.

**Review by DasCapschen — APPROVED** [review]
lgtm

File: src/Resources/SignalTypes/Hp/Form/HpForm.tscn (hunk 1 above)
Comment (DasCapschen) [reply src/Resources/SignalTypes/Hp/Form/HpForm.tscn]: Yes, we need this back in, else the signal animation breaks.

Comment (HaSa1002) [reply src/Resources/SignalTypes/Hp/Form/HpForm.tscn]: fixed

**Pull request merged**